yas3fs/__init__.py

```diff
@@ -927,6 +927,14 @@ def __init__(self, options):
             logger.info("SQS queue name (new): '%s'" % self.sqs_queue_name)
             self.queue.set_message_class(boto.sqs.message.RawMessage) # There is a bug with the default Message class in boto
 
+            self.current_user_aws_principalId = None
+            try:
+                iam = boto.connect_iam()
+                self.current_user_principalId = u'AWS:'+iam.get_user()['get_user_response']['get_user_result']['user']['user_id']
+                logger.info("Current user principalId: "+self.current_user_principalId)
+            except Exception as e:
```

Comment:
We protect this against permission denied exceptions, as existing users may not have granted the "getUser" permission to the user running yas3fs